### ISSUE TITLE
Documentation on how to construct the loss curve

### ISSUE THREAD

**Issue by l-omar-chehab** [benchopt/benchopt#578]
As I understand it, there are three different questions on how to construct the performance curve:

1. Should I **restart** the optimization from scratch at each iteration (n_iter vs. callback)

2. How should I **sample** the iterations (default is exponential increase vs. custom increase using get_next)

3. When should I **stop** the optimizer (default is a SufficientProgressCirterion)

The documentation https://benchopt.github.io/performance_curves.html currently addresses questions (1) and (2) as a package, when in fact they can be chosen separately: you could choose “callback” and also whatever sampling choice, right?

If this is the case, maybe we could clarify this?

**Comment by tomMoral** [comment]
Yes , you can indeed have callback with whatever sampling strategy you want.
It would be nice to add the the option you describe at the beginning of the page on curve construction I think, and man it clear they are independent.
